Source/Core/Core/HW/EXI/EXI.cpp

```diff
@@ -270,7 +270,7 @@ void UpdateInterrupts()
   ProcessorInterface::SetInterrupt(ProcessorInterface::INT_CAUSE_EXI, causeInt);
 }
 
-static void UpdateInterruptsCallback(u64 userdata, s64 cycles_late)
+static void UpdateInterruptsCallback(Core::System& system, u64 userdata, s64 cycles_late)
```

Comment:
Just curious but what was this added for?

Reply:
It won't compile unless the function signature of this callback matches.

Reply:
Can't pass a function pointer unless the signature matches.

Reply:
Thank you both, hadn't gotten that far in my investigation yet. That makes sense!